CMakeLists.txt

```diff
@@ -27,7 +27,7 @@ endif()
 set(CMAKE_C_COMPILER_NAMES gcc clang)
 include_directories("include")
 add_executable(ip2gid
-	src/main.c src/log.c)
+	src/main.c src/log.c src/ip2gid.c src/client.c src/server.c)
```

Comment:
Please do the change in two patches.
1) move server into a separate file.
2) move client into a separate file.